contracts/strategies/StrategyCurveBTCVoterProxy.sol

```diff
@@ -166,6 +166,7 @@ contract StrategyCurveBTCVoterProxy {
             IERC20(want).safeTransfer(IController(controller).rewards(), _fee);
             deposit();
         }
+        VoterProxy(proxy).lock();
```

Comment:
Simply calling `StrategyProxy.lock()` could brick the harvest if the available CRV value is zero. See this line in `VotingEscrow.increase_amount()`:
```
assert _value > 0  # dev: need non-zero value
```

Reply:
I suggest handling this in `StrategyProxy.lock()` method.